installer/parse-install-yaml.py

```diff
@@ -133,6 +147,12 @@ def main():
         command = command.replace(" ", "^")
         commands.append(command)
 
+    for install_type, pkg_list in now_cache.items():
```

Comment:
This change doesn't preserve the order of the `-now` commands in the yaml file

Reply:
That is true, but that shouldn't matter.

Because any item in the list should not depend on any other command in the list.

Reply:
That is true